docs/graphql/manual/auth/authentication/jwt.rst

```diff
@@ -123,7 +123,8 @@ JSON object:
      "key": "<optional-key-as-string>",
      "jwk_url": "<optional-url-to-refresh-jwks>",
      "claims_namespace": "<optional-key-name-in-claims>",
-     "claims_format": "json|stringified_json"
+     "claims_format": "json|stringified_json",
+     "header": <optional-key-to-indicate-cookie-or-authorization-header>
```

Comment:
this will break the code highlighting as the "header" value is not valid json. will need to change the ``code-block:: json`` to ``code-block:: none``